src/components/rounds/matchDays/matchDayForm.tsx

```diff
@@ -30,7 +30,6 @@ const MatchDayForm = ({ roundId, matchDay }: FormProps) => {
 
   const handleFormChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     setFormMatchDay((prevState) => {
-      console.log(e.target.value);
       return {
```

Comment:
Could this display some sort of success msg? In previous version input became green for a moment but toast msg would be fine too